src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,97 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Defines <Code>Storage</Code> class.
+ * <p>In charge of reading and writing files from disk.</p>
+ */
```

Comment:
I like how you added javadocs to all your classes, fields and methods.

Reply:
Thank you for your kind comments!